pkgs/development/tools/dyff/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromGitHub}:
+
+let
+  version = "1.4.3";
+in
+
+buildGoModule rec {
+  inherit version;
+
+  pname = "dyff";
```

Comment:
```suggestion
buildGoModule rec {
  pname = "dyff";
  version = "1.4.3";
```

Reply:
Thank you for your feedback. I have made the suggested changes.